addons/mail/static/src/core/common/chat_hub.js

```diff
@@ -49,6 +49,7 @@ export class ChatHub extends Component {
             }
         });
         useMovable({
+            enable: () => !this.chatHub.opened.length,
```

Comment:
Should make it draggeable when `chatHub.compact`, otherwise the "Hide all conversations" is not draggeable when we expect it should be the case.